CONTRIBUTING.md

```diff
@@ -130,41 +122,51 @@ Issue #190: Short summary of the issue
 ```
 
 Code
-====
+----
 
 This section include some tips that will help you to push source code.
 
-::: {.note}
-::: {.admonition-title}
-Note
-:::
+**Note:**
+> We recommend using a self-contained environment, 
+> such as [Python virtual environments](https://docs.python.org/3/tutorial/venv.html) 
+> or Anaconda environments, to cleanly separate Qiskit from other applications 
+> and improve your experience.
 
-We recommend using [Python virtual
-environments](https://docs.python.org/3/tutorial/venv.html) to cleanly
-separate Qiskit from other applications and improve your experience.
-:::
+### Setup with Conda virtual environment
 
-Setup with an environment
--------------------------
+``` {.sh}
+$ conda create -y -n QiskitDevenv python=3
+$ conda activate QiskitDevenv
+```
 
-The simplest way to use environments is by using Anaconda
+### Setup with Python virtual environment
 
-``` {.sh}
-conda create -y -n QiskitDevenv python=3
-source activate QiskitDevenv
+```{.sh}
```

Comment:
nit - can you add a sentence before each command to make it read more smoothly? maybe something like 

 Issue the following command to create a Python virtual environment:
```
<command>
```